# Room Configuration
rooms:
  living_room:
    name: "Living Room"
    type: "living_room"
    center: {x: 2.0, y: 1.0, z: 0.0}
    area: 20.0
    is_accessible: true
    typical_threats: ["motion_detection", "temperature_anomaly"]
    normal_conditions:
      temperature: 22.0
      humidity: 45.0
      
  kitchen:
    name: "Kitchen"
    type: "kitchen"
    center: {x: 5.0, y: 1.0, z: 0.0}
    area: 15.0
    is_accessible: true
    typical_threats: ["temperature_anomaly", "door_state"]
    normal_conditions:
      temperature: 20.0
      humidity: 50.0
      
  bedroom_1:
    name: "Master Bedroom"
    type: "bedroom"
    center: {x: 2.0, y: 4.0, z: 0.0}
    area: 18.0
    is_accessible: true
    typical_threats: ["motion_detection", "door_state"]
    normal_conditions:
      temperature: 21.0
      humidity: 40.0
      
  bedroom_2:
    name: "Guest Bedroom"
    type: "bedroom"
    center: {x: 5.0, y: 4.0, z: 0.0}
    area: 12.0
    is_accessible: true
    typical_threats: ["motion_detection", "door_state"]
    normal_conditions:
      temperature: 20.0
      humidity: 40.0
      
  bathroom:
    name: "Bathroom"
    type: "bathroom"
    center: {x: 1.0, y: 3.0, z: 0.0}
    area: 8.0
    is_accessible: true
    typical_threats: ["temperature_anomaly", "door_state"]
    normal_conditions:
      temperature: 23.0
      humidity: 65.0
      
  hallway:
    name: "Hallway"
    type: "hallway"
    center: {x: 3.0, y: 2.5, z: 0.0}
    area: 10.0
    is_accessible: true
    typical_threats: ["motion_detection"]
    normal_conditions:
      temperature: 21.0
      humidity: 45.0
      
  entry:
    name: "Entry Way"
    type: "entry"
    center: {x: 0.5, y: 0.5, z: 0.0}
    area: 5.0
    is_accessible: true
    typical_threats: ["motion_detection", "door_state"]
    normal_conditions:
      temperature: 20.0
      humidity: 45.0

# Door configuration
doors:
  front_door:
    name: "Front Door"
    location: {x: 0.0, y: 0.0, z: 0.0}
    expected_distance: 0.5
    tolerance: 0.1
    expected_open: false
    
  back_door:
    name: "Back Door"
    location: {x: 6.0, y: 0.0, z: 0.0}
    expected_distance: 0.5
    tolerance: 0.1
    expected_open: false
    
  bedroom_1_door:
    name: "Master Bedroom Door"
    location: {x: 2.0, y: 3.0, z: 0.0}
    expected_distance: 0.3
    tolerance: 0.1
    expected_open: true
    
  bedroom_2_door:
    name: "Guest Bedroom Door"
    location: {x: 5.0, y: 3.0, z: 0.0}
    expected_distance: 0.3
    tolerance: 0.1
    expected_open: true
    
  bathroom_door:
    name: "Bathroom Door"
    location: {x: 1.0, y: 2.5, z: 0.0}
    expected_distance: 0.3
    tolerance: 0.1
    expected_open: true
